projects/Magnetosphere/Magnetosphere.cpp

```diff
@@ -702,8 +702,16 @@ namespace projects {
       phiprof::Timer refinesTimer {"Set refines"};
       int myRank;       
       MPI_Comm_rank(MPI_COMM_WORLD,&myRank);
-      if (myRank == MASTER_RANK)
+      if (myRank == MASTER_RANK) {
          std::cout << "Maximum refinement level is " << mpiGrid.mapping.get_maximum_refinement_level() << std::endl;
+      }
+      
+      if (!P::useAlpha && !P::useJPerB) {
+         if (myRank == MASTER_RANK) {
+            std::cout << "WARNING All refinement indices disabled" << std::endl;
```

Comment:
Tell the user of the future this means no refinement will take place, or only static if activated? Or will this function not be called for static, so just tell that no refinement is expected to occur?

Reply:
This code path is only if you have adaptive refinement enabled with both refinement indices disabled. Thinking more on it, I think that should be an early return as well

Reply:
Well it is a return just below this.

Reply:
So it is, just with a wrong wrong return type (now fixed)